components/sidebar/sidebar_channel/sidebar_channel_menu/sidebar_channel_menu.tsx

```diff
@@ -225,6 +243,7 @@ export class SidebarChannelMenu extends React.PureComponent<Props, State> {
             <React.Fragment>
                 <Menu.Group>
                     {markAsRead}
+                    {markAsUnread}
```

Comment:
@esethna The spec has Mark as Unread appearing after the Favourite and Mute options, but presumably it makes sense to have at the same spot as Mark as Read. At the same time, I feel like it makes sense to gave Favourite and Mute first.

Thoughts on which way to go with that?

Reply:
I'll let @rmonga71 make the call, you make a good point and think it makes sense at the top given it's a more commonly used action in a users workflow

Reply:
Thanks @esethna for looping me in. Agree with you both - I like Mark as Unread at the top so it's in the same spot as Mark as Read, and I'd expect it's the most common user action from that menu.